lib/Date/Utility.pm

```diff
@@ -931,6 +925,31 @@ sub truncate_to_day {
     return $popular{$tepoch} // Date::Utility->new($tepoch);
 }
 
+=head2 move_to_nth_dow
+
+Returns undef or a Date::Utility object pointing to nth day of week of the current month.
+$dow is a three letter abbreviation of weekday names.
```

Comment:
If one only reads the documentation without looking at the code, one cannot understand how to use the method.